pages/1.13/monitoring/logging/aggregating/splunk/index.md

```diff
@@ -9,129 +9,73 @@ enterprise: false
 ---
 
 # Overview
-You can pipe system and application logs from a DC/OS cluster to your existing Splunk server. This document describes how to configure a Splunk universal forwarder to send output from each node to a Splunk installation. This document does not explain how to set up and configure a Splunk server.
+You can pipe system and application logs from a DC/OS cluster to your existing Splunk server. This document describes how to configure Fluent Bit to send output from each node to a Splunk installation. This document does not explain how to set up and configure a Splunk server.
```

Comment:
Replace `This document describes how to configure Fluent Bit to send output from each node to a Splunk installation. This document does not explain how to set up and configure a Splunk server.`

with `This page describes how to configure Fluent Bit to send output from each node to a Splunk installation. It does not explain how to set up and configure a Splunk server.